src/main/java/net/snowflake/ingest/utils/HttpUtil.java

```diff
@@ -109,17 +110,19 @@ private static void initHttpClient() {
   private static ServiceUnavailableRetryStrategy getServiceUnavailableRetryStrategy() {
     return new ServiceUnavailableRetryStrategy() {
       private int executionCount = 0;
-      int REQUEST_TIMEOUT = 408;
+      final int REQUEST_TIMEOUT = 408;
 
       @Override
       public boolean retryRequest(
           final HttpResponse response, final int executionCount, final HttpContext context) {
         this.executionCount = executionCount;
         int statusCode = response.getStatusLine().getStatusCode();
-        LOGGER.info(
-            "In retryRequest for service unavailability with statusCode:{} and uri:{}",
-            statusCode,
-            getRequestUriFromContext(context));
+        if (statusCode != 200) {
```

Comment:
can we move this to the block where we check `if (needNextRetry)`?

Reply:
We only want to print in retryRequest if we satisfy the criteria for retry..

Reply:
Ok,I thought it was more like a debug log.